log4j-core/src/main/java/org/apache/logging/log4j/core/impl/ThrowableFormatOptions.java

```diff
@@ -298,7 +321,28 @@ public static ThrowableFormatOptions newInstance(String[] options) {
                 }
             }
         }
-        return new ThrowableFormatOptions(lines, separator, packages, ansiRenderer, suffix);
+        return new ThrowableFormatOptions(lines, separator, filterPackages, filterStartFrames, ansiRenderer, suffix);
+    }
+
+    /**
+     * Common code for extracting packages from "filters(com.example)" and "filters.startFrames(com.example)"
+     *
+     */
+    private static List<String> extractFilters(List<String> packages, String option, String conversionWord) {
```

Comment:
If you don't want to get @garydgregory on your neck, you better have `final` modifiers here. :stuck_out_tongue_winking_eye: